spec/classes/mod/security_spec.rb

```diff
@@ -79,13 +79,17 @@
                 audit_log_type: 'Concurrent',
                 audit_log_storage_dir: '/var/log/httpd/audit',
                 secdefaultaction: 'deny,status:406,nolog,auditlog',
+                secrequestbodyaccess: 'On',
```

Comment:
Minor change, but for the test's could you set the values to something other than the default?

Reply:
Changed the values and rebased